### DISCUSSION TITLE
rotate photos album

### DISCUSSION THREAD

**Discussion by doreks** [photoprism/photoprism#1047] [Q&A] [answered]
Hi, 

I am importing some of my photos to this platform and I would like to know if there is a way to rotate the photos when you are viewing them. Maybe it would be a good idea if you could rotate the photo and save this change so that you don't have to rotate it again.

thanks!

**Comment by StephenBrown2** [comment, marked as answer]
See already open issue: https://github.com/photoprism/photoprism/issues/464
